packages/snaps-sdk/src/jsx/components/SettingCell.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  createSnapComponent,
+  type GenericSnapElement,
+  type SnapsChildren,
+} from '../component';
+
+/**
+ * The props of the {@link SettingCell} component.
+ *
+ * @property title - The title.
+ * @property description - The description.
+ * @property children - The children to show within the cell.
+ */
+export type SettingCellProps = {
+  title: string;
+  description: string;
+  children: SnapsChildren<GenericSnapElement>;
+};
+
+const TYPE = 'SettingCell';
+
+/**
+ * A setting cell component which can be used to display a setting control.
+ *
+ * @param props - The props of the component.
+ * @param props.title - The title.
+ * @param props.description - The description.
+ * @param props.children - The children to show within the cell.
+ * @returns A setting cell element.
+ * @example
+ * <SettingCell title="Title" description="Description">
+ *  <Input name="setting1" />
+ * </SettingCell>
+ */
+export const SettingCell = createSnapComponent<SettingCellProps, typeof TYPE>(
```

Comment:
Wondering if this needs a different name.

Any ideas @Mrtenz ?

Reply:
I wonder if we need this at all?

```tsx
<Box>
  <Heading>Setting name</Heading>
  <Text>Description of the setting.</Text>
  <Field>
    <Input ... />
  </Field>
</Box>
```

Reply:
Couldn't we style headings and text differently when it's on the settings page?

Reply:
That seems like it would get complicated somewhat fast. I'm also wondering if it isn't interesting to expose the "settings styling" for use elsewhere, e.g. in a home page?

Reply:
Having a component likely also makes it easier for us to inject smaller settings options from Snaps in the native settings.

It's a tough thing to balance though, because we want to minimize the amount of components for us to maintain.

Reply:
We would only need a prop on `Text` to change the font weight and this component could be build with Snaps UI

![image](https://github.com/user-attachments/assets/e11db7bb-f80f-4e6c-8a28-9fd69b098939)

Reply:
> It's a tough thing to balance though, because we want to minimize the amount of components for us to maintain.

I wonder if at some point we could expose "helpers" Snaps UI components, things that would be written in Snaps UI JSX so they don't need maintenance on the extension side

Reply:
We could, assuming we expose all the necessary styling.

Reply:
Resolved, dropping this component